acceptance/tests/peering/peering_connect_test.go

```diff
@@ -138,9 +157,13 @@ func TestPeering_Connect(t *testing.T) {
 				require.NotEmpty(r, acceptorSecretName)
 			})
 
+			fmt.Println("copying secret from client peer to server peer, printing contents:")
 			// Copy secret from client peer to server peer.
 			k8s.CopySecret(t, staticClientPeerClusterContext, staticServerPeerClusterContext, "api-token")
+			resp, _ := k8s.RunKubectlAndGetOutputE(t, staticClientPeerClusterContext.KubectlOptions(t), "secret", "api-token", "-o", "yaml")
+			fmt.Println(resp)
```

Comment:
Might want to check the syntax on this one.. since we're not using `resp` anywhere but the println?
```suggestion
			k8s.RunKubectl(t, staticClientPeerClusterContext.KubectlOptions(t), "secret", "api-token", "-o", "yaml")
```

Reply:
Removed these lines instead, they actually weren't working